### PR TITLE
feat(universal-header): adjust desktop category style & update icons

### PR DESCRIPTION
**Pull request by Aylie-Chou**
- add util/icons.js to load icon file
- remove mobile/tablet style in desktop only component (header, icons, channel)
- new category style

related to [TWREPORTER-7](https://twreporter-org.atlassian.net/browse/TWREPORTER-7)

### CODE REVIEW

File: packages/universal-header/src/containers/header.js [hunk 1]
```diff
@@ -193,9 +167,12 @@ class Container extends React.PureComponent {
         <MobileOnly>
           <MobileHeader menu={mobileMenu} {...passThrough} />
         </MobileOnly>
-        <NonMobileOnly>
+        <TabletOnly>
```
Comment (taylrj) [inline packages/universal-header/src/containers/header.js:170]: Why do we need `<TabletOnly>` if its child (i.e. `<MobileHeader>`) has no difference with `<MobileOnly>` ?

**Review by taylrj — COMMENTED** [review]
Should we remove `packages/universal-header/static/photography/.DS_Store`, `packages/universal-header/static/normal/.DS_Store` ?

File: packages/universal-header/src/components/icons.js [hunk 2]
```diff
@@ -30,6 +30,12 @@ const IconsContainer = styled.div`
   `}
 `
 
+const ShowOnHover = styled.div`
+`
```
Comment (taylrj) [inline packages/universal-header/src/components/icons.js:34]: How about moving the hover selector into `ShowOnHover` and `HideOnHover` ?

such as:

```suggestion
const ShowOnHover = styled.div`
  display: none;
  &:hover {
    display: block;
  }
`
```

Is it doable ?

File: packages/universal-header/src/components/channels.js [hunk 3]
```diff
@@ -64,99 +54,104 @@ const dropDownMenuEffectCSS = css`
...
   display: flex;
   flex-direction: row;
   flex-wrap: nowrap;
   align-items: center;
   list-style-type: none;
-  margin: 0;
+  margin: auto;
+`
+
+const ShowOnHover = styled.div`
```
Comment (taylrj) [inline packages/universal-header/src/components/channels.js:101]: How about moving the hover selector into `ShowOnHover` and `HideOnHover` ?

File: packages/universal-header/src/components/drop-down-menu.js [hunk 4]
```diff
@@ -18,17 +18,9 @@ const _ = {
...
-  contentMaxWidth: 320, // px
-  containerPadding: {
-    desktop: [10, 0], // px
-  },
   itemPadding: {
-    mobile: [30, 27], // px
-    tablet: [30, 33], // px
-    desktop: [9, 22], // px
-    hd: [9, 46], // px
+    desktop: [12, 16], // px
```
Comment (taylrj) [inline packages/universal-header/src/components/drop-down-menu.js:22]: Do we still need to specify `desktop` if the style is shared among other devices ?

File: packages/universal-header/src/components/icons.js (hunk 2 above)
Comment (Aylie-Chou) [reply packages/universal-header/src/components/icons.js:34]: The reason to put :hover in their parent is that the hover to show/hide effect should be triggered even when its parent been hoverd.
I think current method would make code a little bit dazzled.
Please let me know if you come out a better way for it.

File: packages/universal-header/src/containers/header.js (hunk 1 above)
Comment (Aylie-Chou) [reply packages/universal-header/src/containers/header.js:170]: The child of TabletOnly will be different in the next PR XD
Should I combine MobileOnly & TabletOnly in this version ?

Comment (taylrj) [reply packages/universal-header/src/containers/header.js:170]: No, just keep it as it is. I'll resolve this.

File: packages/universal-header/src/components/icons.js (hunk 2 above)
Comment (taylrj) [reply packages/universal-header/src/components/icons.js:34]: Our use case seems like [referring-to-other-components](https://styled-components.com/docs/advanced#referring-to-other-components).

Would it be acceptable to you ?

Comment (Aylie-Chou) [reply packages/universal-header/src/components/icons.js]: Looks great.
update it in commit 7478483

File: packages/universal-header/src/components/drop-down-menu.js (hunk 4 above)
Comment (Aylie-Chou) [reply packages/universal-header/src/components/drop-down-menu.js]: Remove it in commit 7478483.

**Comment by taylrj** [comment]
LGTM

**Pull request merged**